third_party/VENDOR-LICENSE

```diff
@@ -5402,6 +5402,33 @@ Import: github.com/tektoncd/pipeline/vendor/github.com/modern-go/reflect2
 
 
 
+===========================================================
```

Comment:
uh? why this change?

Reply:
@afrittoli not sure, ran `./hack/update-codegen.sh` and it added that :sweat: